### PR TITLE
Add minutes for TLT meetings for 3 September and 1 October 2024

### PR DESCRIPTION
**Pull request by valeriupredoi**
@schlunma you were the minute taker for the 3 Sept call, but I don't see any minutes, was that call just general banter? :grin: At any rate, I added an entry and some generic minutes, just to show we had it :beer:

### CODE REVIEW

**Comment by schlunma** [comment]
Ah, right, forgot about that. 

I included most of the points we discussed in the action items, do you want to add those was well?

**Comment by valeriupredoi** [comment]
just did, Manu :beer:

File: Tech Lead/Minutes/20241001.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- @schlunma: What do we about NCL? There are (again) problems with our environment, see [ESMValGroup/ESMValCore#2535](https://github.com/ESMValGroup/ESMValCore/issues/2535): separate module maybe; @bouweandela: leave it to user to install it; @schlunma: diagnostics are old, most devs switching to Python for new stuff -> **topic to be brought at the workshop**; @rbeucher: a few recipes (eg Perfmetrics) that are good, to be ported to Python: possible solutions include separate conda envs and docker containers; this item has a **Workshop TODO**;
+- @schlunma: Very brief update on ESMValTool+Dask paper: to be re-circulated today(ish) and some of us still need to send comments; @bouweandela give feedback on current performance status: few benchmark discrepancies to look at ie perfomance is *even* better (eg used dev branch of `iris`);
```
Comment (schlunma) [inline Tech Lead/Minutes/20241001.md:30]:
```suggestion
- @schlunma: Very brief update on ESMValTool+Dask paper: to be re-circulated today(ish) and some of us still need to send comments; @bouweandela give feedback on current performance status: few benchmark discrepancies to look at, i.e. perfomance is *even* better (e.g. used dev branch of `iris`);
```

File: Tech Lead/Minutes/20241001.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- @valeriupredoi: work on modernizing and re-deploying our ESMValBot https://github.com/ESMValGroup/ESMValBot/pull/37
+- @valeriupredoi: `codacy-ruff` plugin to Codacy: https://github.com/ESMValGroup/ESMValTool/issues/3739
+- @rbeucher: `intake-esm` catalog support: more and more interest in adding datasets to the ESM catalog; @bouwenadela: looking at XCube also, a proposal in the pipes, so there may be time to implement `intake` support; coordinate with Tina, so there is no duplication -> a meeting to be set up;
+- @schlunma wants to merge PR that introduces new (dask-like) configuration system today ([ESMValGroup/ESMValCore#2448](https://github.com/ESMValGroup/ESMValCore/pull/2448)): this needs ne merged **ASAP**;
```
Comment (schlunma) [inline Tech Lead/Minutes/20241001.md:34]:
```suggestion
- @schlunma wants to merge PR that introduces new (dask-like) configuration system today ([ESMValGroup/ESMValCore#2448](https://github.com/ESMValGroup/ESMValCore/pull/2448)): this needs be merged **ASAP**;
```

**Review by schlunma — APPROVED** [review]
Thanks V, just some tiny comments from my side!

**Pull request merged**